core/input/input.cpp

```diff
@@ -286,7 +286,7 @@ bool Input::is_action_pressed(const StringName &p_action, bool p_exact) const {
 	return action_state.has(p_action) && action_state[p_action].pressed && (p_exact ? action_state[p_action].exact : true);
 }
 
-bool Input::is_action_just_pressed(const StringName &p_action, bool p_exact) const {
+bool Input::is_action_just_pressed(const StringName &p_action, bool p_exact, bool p_current) const {
```

Comment:
I can't see the difference between calling this with `p_current = true` and calling `is_action_pressed()`.